swift3/response.py

```diff
@@ -107,6 +107,8 @@ def __init__(self, *args, **kwargs):
                           'etag', 'last-modified', 'x-robots-tag',
                           'cache-control', 'expires', 'x-amz-version-id'):
                 headers[key] = val
+            elif _key == 'x-timestamp':
+                headers['x-amz-version-id'] = val
```

Comment:
Using the current timestamp as a version ID may yield inconsistent results. Can't we use `x-object-sysmeta-version-id` or at least `last-modified` (which seems to be set by oio-swift as the object's creation time).